tests/core/core_meta_test.py

```diff
@@ -121,13 +121,32 @@ def body(scope, x):
   def test_get_partition_spec(self):
     xs = {'kernel': meta.Partitioned(jnp.zeros((8, 3, 3)),
                                      ('layers', 'in', 'out')),
-          'bias': jnp.zeros((8, 3))}
+          'bias': jnp.zeros((8, 3)),
+          'step': jnp.array(100)}
     ps = meta.get_partition_spec(xs)
     self.assertEqual(
         ps,
         {
             'kernel': jax.sharding.PartitionSpec('layers', 'in', 'out'),
-            'bias': None,
+            'bias': jax.sharding.PartitionSpec(),
+            'step': jax.sharding.PartitionSpec(),
+        },
+    )
+
```

Comment:
Consider adding a test for `logical_to_mesh_sharding`

Reply:
Done in `partitioning_test.py`